hydromt/data_adapter/geodataset.py

```diff
@@ -285,6 +299,12 @@ def _read_data(self, fns, logger=logger):
         else:
             raise ValueError(f"GeoDataset: Driver {self.driver} unknown")
 
+        if len(ds) == 0:
+            if handle_nodata == NoDataStrategy.RAISE:
+                raise NoDataException(f"No data was read from files {fns}")
+            elif handle_nodata == NoDataStrategy.IGNORE:
+                logger.warn(f"no data was found at {fns}. skipping...")
```

Comment:
Just a general question, I thought it was better to use `logger.warning` in stead of `logger.warn`. Is there a difference or preference here?